examples/robot-connector/robot.py

```diff
@@ -0,0 +1,184 @@
+"""
+This file represents the collection of API wrappers that fetch information from the
+robot.
+"""
+
+import asyncio
+import random
+from typing import Coroutine
+
+
+class ExampleBotAPIWrapper:
+    """
+    This class contains an async wrapper for the robot's API.
+    Real API endpoints would be hit here.
+    """
+
+    def __init__(self, endpoint: str, api_key: str):
+        self.endpoint = endpoint
+        self.api_key = api_key
+
+    async def fetch_telemetry_data(self) -> dict:
```

Comment:
Using `dict` as return type may be convenient, but that way we can't enforce the exact fields. Wouldn't it be better to for certain methods to be more strict so we minimize the chances of typos in dict keys ?